tensorflow_addons/layers/python/wrappers.py

```diff
@@ -13,17 +13,21 @@
 # limitations under the License.
 # =============================================================================
 
+import tensorflow as tf
+
 from tensorflow import name_scope
-from tensorflow.python.framework import ops
 from tensorflow.python.framework import tensor_shape
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import nn_impl
+from tensorflow.python.ops import variables as tf_variables
+from tensorflow.python.ops.nn import moments
+from tensorflow.python.ops.math_ops import sqrt
+from tensorflow.python.ops.linalg_ops import norm
+
 from tensorflow.python.keras import initializers
-from tensorflow.python.eager import context
-from tensorflow.python.keras.engine.base_layer import Layer
-from tensorflow.python.keras.engine.base_layer import InputSpec
+from tensorflow.python.keras.engine.base_layer import Layer, InputSpec
```

Comment:
I prefer to put them [on separate lines](https://www.python.org/dev/peps/pep-0008/#imports), or `from tensorflow.python.keras.engine import base_layer`. What do you think?

Reply:
Agree, I pushed an update using base_layer